components/resources/commands.grdp

```diff
@@ -443,6 +443,9 @@
   <message name="IDS_IDC_SHOW_WAYBACK_MACHINE_BUBBLE">
     Show Wayback Machine bubble
   </message>
+  <message name="IDS_IDC_NEW_EMAIL_ALIAS">
```

Comment:
hey, you override this in `generate_command_metadata.py`, so I think its safe to remove

Reply:
I think this is actually needed? (Distinct from `IDS_IDC_SHOW_EMAIL_ALIAS`.)